### PR TITLE
Favicon + manifest

### PR DESCRIPTION
**Pull request by hsab**

### CODE REVIEW

File: public/manifest-assets/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "name": "NEAR.REST",
```
Comment (blerchin) [inline public/manifest-assets/site.webmanifest:2]: Shouldn't this be NEARREST NEIGHBOR?

File: public/manifest-assets/html_code.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
```
Comment (blerchin) [inline public/manifest-assets/html_code.html:1]: What is this? looks like copypasta

**Review by blerchin — APPROVED** [review]

File: public/manifest-assets/site.webmanifest (hunk 1 above)
Comment (hsab) [reply public/manifest-assets/site.webmanifest:2]: This when added to home page. I can easily change, but thought NEAR.REST would be more appropriate. Also NEARREST NEIGHBOR looked a bit too long on my homescreen compared to everything else.

File: public/manifest-assets/html_code.html (hunk 2 above)
Comment (hsab) [reply public/manifest-assets/html_code.html:1]: https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html

**Pull request merged**